craco.config.cjs

```diff
@@ -100,10 +125,35 @@ module.exports = {
           return true
         })
 
-      // We're currently on Webpack 4.x which doesn't support the `exports` field in package.json.
-      // Instead, we need to manually map the import path to the correct exports path (eg dist or build folder).
-      // See https://github.com/webpack/webpack/issues/9509.
-      webpackConfig.resolve.alias['@uniswap/conedison'] = '@uniswap/conedison/dist'
+      webpackConfig.resolve.plugins = webpackConfig.resolve.plugins.map((plugin) => {
+        // Allow vanilla-extract in production builds.
+        // See https://sandroroth.com/blog/vanilla-extract-cra#production-build.
+        if (plugin instanceof ModuleScopePlugin) {
+          plugin.allowedPaths.push(path.join(__dirname, 'node_modules/@vanilla-extract/webpack-plugin'))
+        }
+
+        return plugin
+      })
+
+      const rules = webpackConfig.module.rules[1].oneOf
```

Comment:
nit: This section is missing a high-level explanation of what it does. I assume it has something to do with `@uniswap/analytics` package, but I would appreciate additional explanation.